pagopa/io-app
# chore(IT Wallet): [SIW-1440] Add Disability Card credential issuance

## Short description
This PR adds the Disability Card details in the credential details screen, such as the QR Code and the ability to display boolean claims.
It also contains small improvements and unused code removals, as listed in the changes below.

## List of changes proposed in this pull request
- Fixed `useItwDisableGestureNavigation` hook name
- Added static logo and name for the credential's issuer in the `ItwIssuanceCredentialTrustIssuerScreen` component
- Added `ItwGenericErrorComponent` which displays a generic failure and used it instead of `costNull`
- Added `BoolClaim` decoder and `BoolClaimItem` component, which renders a "Yes/No" claim in the credential preview/details screen
- Added `UrlClaim` decoder to decode URL type claims
- Added `QrCodeImage` component to display a QR Code image from a string value
- Added `ItwQrCodeClaimImage` component, which wraps `QrCodeImage` for the ITW QR Code claims
- Added `WellKnownClaim` enum, which describes the claims that we need to manipulate manually in order to render the correct information
- Added `getHumanReadableParsedCredential`, which helps to print data in a human readable format
- Added credential issuance buttons to ITW playgrounds
- Changed `ignoreMissingAttributes` parameter to `false` in the `verifyAndParseCredential` function call during the issuance of a credential: this will let us know if the credential claims does not conform to the received EC
- Refactored `ItwCredentialsClaimsSection` component: 
  - it has been moved to the `presentation` folder and it now uses `ItwCredentialClaimsList` component
  - it can display a QR Code if the credential has one, using the new `ItwQrCodeClaimImage` component 
- Moved `ItwCredentialClaimsSection` from `common` to `presentation` folder
- Removed everything related to claims section, which are no more needed: `ClaimSection`, `sectionsByClaim`, `groupCredentialClaims`
- Removed `itwErrorUtils.ts` utilities
- Removed `useItwIdpIdentification.ts` hook

## How to test
In the PRE env, start the Disability Card issuance from the IT Wallet Playground. You should be able to obtain the credential and see it in your wallet.
